R/update.R

```diff
@@ -191,7 +191,11 @@ docdb_update.src_sqlite <- function(src, key, value, ...) {
                               WHERE key = '%s'
                                 AND value = %s;",
                         key, key,
-                        vn[1], value[i , 1]))
+                        vn[1], 
+                        ifelse(class(value[i, 1]) == "character", 
```

Comment:
would `inherits(value[i, 1], "character")` be better? in case there are two classes for some reason, which would at least raise a warning here

Reply:
sure, thanks, now in commit 72e4441